### PR TITLE
Form

### PR DESCRIPTION
**Pull request by mamatkulov95**
Created Form section and made it intreactive.
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import "./App.css";
```
Comment (abanseka) [inline src/App.js:1]: CSS imports should come last. 
> import hierarchy

1. Third-party packages or modules
2. Components or modules
3. CSS files

File: src/Form/Form.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  font-size: 16px;
+}
+
+.meme {
+  position: relative;
+}
+
+.meme-image {
+  width: 100%;
+  /* max-height: 60vh; */
```
Comment (abanseka) [inline src/Form/Form.css]: Ensure not to push irrelevant comments

File: src/Form/Form.css [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+.meme-text {
+  position: absolute;
+  width: 80%;
+  text-align: center;
+  left: 50%;
+  transform: translateX(-50%);
+  margin: 15px 0;
+  padding: 0 5px;
+  font-family: impact, sans-serif;
+  font-size: 2em;
```
Comment (abanseka) [inline src/Form/Form.css]: Ensure consistent usage of units.
if `px` then `px` likewise with `em` Not Both

File: src/Form/Form.css [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    2px 2px 5px #000;
+}
+
+.bottom {
+  bottom: 15px;
+}
+
+.top {
+  top: 15px;
+}
```
Comment (abanseka) [inline src/Form/Form.css]: `.bottom` and  `.top` used are ambiguous.
Please try to be specific. `.bottom` and `.top` what?

File: src/Form/Form.js [hunk 5]
```diff
@@ -0,0 +1,64 @@
+import "./Form.css";
+import React, { useState } from "react";
+
+export default function Form() {
+  const [meme, setMeme] = React.useState({
```
Comment (abanseka) [inline src/Form/Form.js]: The use of `React.useState` is overkill.
- `useState` has already been imported from react 
> Review object destructuring
-  use `const [meme,setMeme] = useState({...})`
- You can remove the React import. JSX is automatically interpreted with the latest react release
> `import {useState,useEffect} from "react"`

- [x]  Replace all instances of `React.method()` with just the `method()`

File: src/Form/Form.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  const [allMemes, setAllMemes] = React.useState([]);
+  React.useEffect(() => {
+    fetch("https://api.imgflip.com/get_memes")
+      .then((res) => res.json())
+      .then((data) => setAllMemes(data.data.memes));
+  }, []);
+  function getMemeImage() {
+    const randomNumber = Math.floor(Math.random() * allMemes.length);
+    const url = allMemes[randomNumber].url;
+
```
Comment (abanseka) [inline src/Form/Form.js]: - [x]  Include some spacing and group related code lines. for improved readability

File: src/Form/Form.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+          name="topText"
+          value={meme.topText}
+          onChange={handleChange}
+        />
+        <input
+          type="text"
+          className="input-bottom"
+          name="bottomText"
+          value={meme.bottomText}
+          onChange={handleChange}
```
Comment (abanseka) [inline src/Form/Form.js]: It's essential to guide your users to the action you require them to take.

- [x] Give the inputs some placeholders

**Review by abanseka — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (mamatkulov95) [reply src/App.js:1]: Resolved

File: src/Form/Form.css (hunk 3 above)
Comment (mamatkulov95) [reply src/Form/Form.css]: Changed to px

File: src/Form/Form.css (hunk 4 above)
Comment (mamatkulov95) [reply src/Form/Form.css]: changed.

File: src/Form/Form.css (hunk 2 above)
Comment (mamatkulov95) [reply src/Form/Form.css]: Removed.

File: src/Form/Form.js (hunk 5 above)
Comment (mamatkulov95) [reply src/Form/Form.js]: Removed and Changed according to request

**Review by abanseka — APPROVED** [review]

**Pull request merged**